OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/SpeedAdapter.java

```diff
@@ -0,0 +1,61 @@
+package org.edx.mobile.view.adapters;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.TextView;
+
+import org.edx.mobile.R;
+import org.edx.mobile.core.IEdxEnvironment;
+
+import java.util.Locale;
+
+
+public abstract class SpeedAdapter extends BaseListAdapter<Float> {
```

Comment:
Can we somehow combine this and `ClosedCaptionAdapter`?

Reply:
Yes, We can combine both the adapters but this take more effort than it's worth.

Reply:
got it.